website/assets/styles/components/bubble.component.less

```
@@ -7,24 +7,52 @@
+    &.observer-plus {
+      background-color: rgba(61, 182, 123, 20%);
+      border-color: rgba(61, 182, 123, 20%);
+    }
+    &.gitops {
+      background-color: rgba(61, 182, 123, 20%);
+      border-color: rgba(61, 182, 123, 20%);
+    }
+  }
+
+  &.role-link {
+    .transition(all 250ms);
```

Stylelint (17.7.0)

[error] Unknown word .transition (CssSyntaxError)

(CssSyntaxError)